stage1/aci/aci-manifest.in

```diff
@@ -30,8 +30,12 @@
             "value": "/gc"
         },
         {
+            "name": "coreos.com/rkt/stage1/stop",
+            "value": "@RKT_STAGE1_STOP@"
+        },
+        {
             "name": "coreos.com/rkt/stage1/interface-version",
-            "value": "2"
+            "value": "3"
```

Comment:
ditto for the change 2->3 as discussed in https://github.com/coreos/rkt/pull/2438#discussion-diff-65917341